app/serializers/workflow_serializer.rb

```diff
@@ -49,4 +49,8 @@ def content
   def retirement
     @model.retirement_with_defaults
   end
+
+  def finished_at
+    @model.finished_at || ""
```

Comment:
Why is an empty string cleaner than a null? Is that just because the parser in JS-land is stupid?